setup.py

```diff
@@ -14,5 +14,5 @@
     author_email="nturner@zetta.ai",
     url="https://github.com/ZettaAI/kombu-worker",
     packages=setuptools.find_packages(),
-    install_requires=["kombu", "tenacity", "requests"],
+    install_requires=["kombu", "tenacity", "requests", "task-queue"],
```

Comment:
I don't necessarily want to require `task-queue` because there are ways to use this without it. We can make separate installations if we want instead, but `task-queue` brings in a lot of other dependencies with it.